cms/templates/error.html

```diff
@@ -10,21 +10,35 @@
   % endif
 </%block>
 
+<%!
+from django.conf import settings
+
+if settings.TENDER_DOMAIN:
+  tender_link_start = '<a href="//{domain}/discussion/new" class="show-tender" title="{title}">'.format(
+    domain=settings.TENDER_DOMAIN,
+    title=_("Use our feedback tool, Tender, to share your feedback")
+  ),
+  tender_link_end = '</a>'
+else:
+  tender_link_start = ""
+  tender_link_end = ""
+%>
+
 <%block name="content">
   <article class="error-prompt">
     % if error == '404':
       <h1>${_("The Page You Requested Page Cannot be Found")}</h1>
       <p class="description">${_("We're sorry. We couldn't find the {studio_name} page you're looking for. You may want to return to the {studio_name} Dashboard and try again. If you are still having problems accessing things, please feel free to {link_start}contact {studio_name} support{link_end} for further help.").format(
           studio_name=settings.STUDIO_SHORT_NAME,
-          link_start='<a href="http://help.edge.edx.org/discussion/new" class="show-tender" title="{title}">'.format(title=_("Use our feedback tool, Tender, to share your feedback")),
-          link_end='</a>',
+          link_start=tender_link_start,
+          link_end=tender_link_end,
```

Comment:
Correct. Instead of something like this:
```
unlinked text <a href="foo">linked text</a> unlinked text
```
It will end up like this:
```
unlinked text linked text unlinked text
```
The text will still be there, but there won't be any link. Users won't be able to click on it. I'm open to other suggestions for how to handle this case, but this was the best I could think of.